README.md

```diff
@@ -112,18 +104,21 @@ to the debug console and to the `java/seeq-link-sdk-debugging-agent/build/log/jv
 
 ## Deploying your Connector
 
-When you are ready to deploy your connector to a production environment, execute the `build` command. A zip file will be
+When you are ready to deploy your connector to a test or production remote agent, execute the `build` command. A zip file will be
 created in the `build/distributions` folder of your connector.
 
-Copy this zip file to the Seeq Server you wish to deploy it to. Shut down the server and extract the contents of the zip
-file into the `plugins/connectors` folder within Seeq's `data` folder. (The data folder is usually
-`C:\ProgramData\Seeq\data` on Windows and `~/.seeq/data` on Ubuntu and OSX.) You should end up with one new folder in
-`plugins/connectors`. For example, if you kept the default name for the connector, you would have a
-`plugins/connectors/mycompany-seeq-link-connector-myconnector` folder with a jar file inside.
+1. Shut down the Seeq Remote Agent - execute `seeq stop` in the Seeq CLI
+1. Verify that you have enabled support for plugins
+   1. execute `seeq config get Features/Plugins/Enabled` in the Seeq CLI
```

Comment:
I don't think this is needed. IMHO Features/Plugins/Enabled is only used for plugins (like paralel coordinates for examples). The remote agent is looking into plugins/connectors anyway.

Reply:
I think you're right Marius, I didn't know this!